src/settings-ui/Settings.UI/SettingsXAML/Views/GeneralPage.xaml

```diff
@@ -233,6 +233,18 @@
                 </controls:SettingsGroup>
 
                 <controls:SettingsGroup x:Uid="Appearance_Behavior" IsEnabled="True">
+                    <InfoBar
+                        x:Uid="GPO_SomeSettingsAreManaged"
```

Comment:
Please use bar for single setting and move after runAs settings card.

Or do we plan to implement more settings in this area?

Reply:
I've used the single setting instead. Thank you!